config.py

```diff
@@ -268,6 +268,14 @@ def validate_config(CONFIG: CONFIG_DICT_TYPE) -> None:
             config_assert(online_section.get("move_quality") != "suggest" or not online_section.get("enabled"),
                           f"XBoard engines can't be used with `move_quality` set to `suggest` in {subsection}.")
 
+    for section, subsection in (("online_moves", "online_egtb"),
+                                ("lichess_bot_tbs", "syzygy"),
+                                ("lichess_bot_tbs", "gaviota")):
+        online_section = (CONFIG["engine"].get(section) or {}).get(subsection) or {}
+        if online_section.get("move_quality") == "good":
```

Comment:
Do we want to print the warning if `online_section.get("enabled") == False`?

Reply:
The default value is `best`, so they must have changed it to `good`. I think it is better to warn them, because they could enable it in the future and get unexpected results/errors.